### PR TITLE
Update license_finder information, remove "lpx" module use for Heroku

### PR DESCRIPTION
**Pull request by lfittl**
See individual commit messages.

### CODE REVIEW

File: input/system/heroku/http_handler.go [hunk 1]
```diff
@@ -21,12 +22,83 @@ func SetupHttpHandlerLogs(ctx context.Context, wg *sync.WaitGroup, globalCollect
 	go func() {
 		http.HandleFunc("/", util.HttpRedirectToApp)
 		http.HandleFunc("/logs/", func(w http.ResponseWriter, r *http.Request) {
-			lp := lpx.NewReader(bufio.NewReader(r.Body))
-			for lp.Next() {
-				procID := string(lp.Header().Procid)
-				if procID == "heroku-postgres" || strings.HasPrefix(procID, "postgres.") {
+			reader := bufio.NewReader(r.Body)
```
Comment (keiko713) [inline input/system/heroku/http_handler.go:25]: So... I'm personally curious. Where does this logging format coming from?

**Review by keiko713 — APPROVED** [review]
One question/comment but overall looking good 👍

File: input/system/heroku/http_handler.go (hunk 1 above)
Comment (lfittl) [reply input/system/heroku/http_handler.go:25]: Heroku Logplex :)

Comment (keiko713) [reply input/system/heroku/http_handler.go:25]: Ahh, I was missing the first part of getting `lenStr` and assuming that `83 <40>1 2012-11-30T06:45:29+00:00 host app web.3 - State changed from starting to up` is not in the same format (also I wasn't able to find MSGID but I guess that's `-`).

https://devcenter.heroku.com/articles/log-drains#https-drains

Also it's almost the same as https://datatracker.ietf.org/doc/html/rfc5424 (as it's mentioned in above doc)

**Review by msakrejda — APPROVED** [review]
Looks like CI is failing on lint; otherwise, looks good.

**Pull request merged**